wilson/classes.py

```diff
@@ -90,12 +91,14 @@ class Wilson(ConfigurableClass):
                         'mb_matchingscale': 4.2,
                         'mc_matchingscale': 1.3,
                         'parameters': {},
+                        'wet_method': 'adms'
                         }
 
     # option schema:
     # Voluptuous schema defining allowed option values/types
     _option_schema = vol.Schema({
         'smeft_accuracy': vol.In(['integrate','leadinglog']),
+        'wet_method': vol.In(['betafunctions','adms']),
```

Comment:
Since the new `wet_method` would define a new option that is exposed to the user, I think we should think a bit about how we can extend the interface in a way that makes it possible to later combine the SMEFT and WET options without changing this interface again.

My suggestion would be to use a name for the option that can, at a later point, be used for both WET and SMEFT, e.g. something like `rge_solution` or `rge_method` instead of `wet_method`.

For the options themselves, I would stick to the names that are already used so far. I.e. instead of `'betafunctions'` I would use `'integrate'` (making it possible to also add a `'leadinglog'` option that is based on the betafunctions as well). Since `'integrate'` and `'leadinglog'` specify the way the RGEs are solved rather than the way they are represented (in terms of either betafunctions or adms), I would suggest to use a name describing the solution by an evolution matrix instead of using `'adms'`. Maybe we could use `'matrix'` to keep it short, or `'evolutionmatrix'` or `'evolution_matrix'`.

Any comments on these suggestions and better ideas are also very welcome from @jasonaebischerGIT, @dvandyk, and @DavidMStraub.

Reply:
@peterstangl sure we can invent better names, for the moment I just gave any tentative names, since the PR is currently work under progress. I have not even decided on the names etc and for the moment my focus is to just  produce correct solutions to new beta functions #94. 

I have already implemented `leadinglog` and `integrate` option for WET and will update it soon. 

It would be great idea to go through it in details once the implementation is completed.